standard/variables.md

```diff
@@ -150,6 +150,23 @@ A local variable introduced by a *local_variable_declaration* is not automatical
 >
 > *end note*
 
+#### §discards-new-clause Discards
+
+A ***discard*** is a local variable that has no name and can thus only be referred to once, at the point where it is introduced. A discard is not initially assigned, so it is always an error to access its value. A discard is introduced by a *declaration_expression* with the identifier `_`, and by use of the *simple_name* `_` when lookup of that name does not find a declaration ([§11.7.4](expressions.md#1174-simple-names)).
```

Comment:
Meeting suggestion: Add a *note* to point out that `_` is a valid identifier, and is a ***discard*** only when lookup doesn't find a variable `_`.

Reply:
Note added in upcoming commit